src/consensus/params.h

```diff
@@ -107,6 +107,9 @@ struct Params {
     int nHeightv451Fork;
     int nHeightv452Fork;
 
+    /** Coinbase transaction outputs can only be spent after this number of new blocks (network rule) */
+    int coinbaseMaturity;
```

Comment:
in this struct variable names of type int are prefixed with n in camel case

Reply:
I've amended the commit :D